packages/kbn-test/package.json

```diff
@@ -14,6 +14,7 @@
     "@kbn/babel-preset": "1.0.0",
     "@kbn/dev-utils": "1.0.0",
     "@types/parse-link-header": "^1.0.0",
+    "@types/puppeteer": "^2.1.0",
```

Comment:
why not `3.0.0` to match `puppeteer ` major version?

Reply:
ah, they  published 3.0.0 2 weeks ago, I will give it a try